service/experiment-requests.yaml

```diff
@@ -0,0 +1,4 @@
+- experiment: 2020-06-08
```

Comment:
There was no need to run this experiment. I just did it to test my code and since it provides a decent example (I think examples are as important as docs since many will copy paste).